lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
We weren't planning on adding namespacing to the library until we migrate wholly to the `new-lib` branch. Is there away to remove the need for the `require_once` statements _without_ namespacing? If so, a link to a reference would be super-helpful.

Thanks again for your comments.